### ISSUE TITLE
[DQA] trouble merging issues 

### ISSUE THREAD

**Issue by writetoritu** [PEDSnet/tools#95]
Hi @bruth 
Please refer to the DQA master branch to get the data for the following query: 
```
./pedsnet-dqa merge-issues --token=XXX --program=./ConflictResolution/resolve.py --resolvers=./ConflictResolution/resolvers CCHMC/ETLv8/ ../../CCHMC/DataQualityAssessment/v2.3_Nov2016/issues/
```

When I try this, it says "No new issues found." - which is not true. 

Thanks for your help!

**Comment by bruth** [comment]
Ah. The command takes one or more files, not a directory. So you need to add this `*.csv` at the end, e.g.

```
./pedsnet-dqa merge-issues --token=XXX --program=./ConflictResolution/resolve.py --resolvers=./ConflictResolution/resolvers CCHMC/ETLv8/ ../../CCHMC/DataQualityAssessment/v2.3_Nov2016/issues/*.csv
```

The usage info for the tool states this, but I will update the README as well. I will update the tool to handle this case so either will work.

**Comment by bruth** [comment]
Fixed in 95dfbd7
